### DISCUSSION TITLE
Question about Sensor Watch Builder

### DISCUSSION THREAD

**Discussion by BrianBinFL** [joeycastillo/Sensor-Watch#271] [Q&A] [answered]
I'm new to this project, but not new to programming and things like this. The various pages for this project indicate that firmware must be built for the correct "color" board. There are pre-built images for green boards and blue boards. And the page for the new red boards (which is what I've ordered) indicate that a flag would need to be passed at compile time to target the red board.

But if you cheat and use the [Sensor Watch Builder](https://sensor-watch-builder.fly.dev/), which board does it build for? I realize that the file you create gets run in the emulator, but you can also download the uf2 file which implies that you can use that to flash a board.

My apologies if this is a dumb question or answered elsewhere. I did search. Hopefully I searched well. Thanks in advance for the help.

**Comment by joeycastillo** [comment, marked as answer]
We’ve just been discussing this on the discord! The answer is that right now it builds for the green boards, but once I finalize the flag for building for red boards the plan is to add it to the builder as an option.

**Reply by BrianBinFL** [reply]
Thanks!!!

**Reply by wryun** [reply]
Should work now, by the way, after a few hiccoughs (a combination of my mistakes and not having a board to test on)

**Reply by BrianBinFL** [reply]
Yep, it's working great. I think I've finally settled on the build I'm going to run for a while.

COLOR=RED; SIGNAL_TUNE_DEFAULT=1; simple_clock_face; world_clock_face; sunrise_sunset_face; moon_phase_face; stock_stopwatch_face; alarm_face; thermistor_readout_face; voltage_face; set_time_face; preferences_face; finetune_face; nanosec_face; (8)
